File: dayu/MergeSortedArrayII.java
import java.util.*;
public class MergeSortedArrayII {
    /**
     * @param A and B: sorted integer array A and B.
     * @return: A new sorted integer array
     */
    public ArrayList<Integer> mergeSortedArray(ArrayList<Integer> A, ArrayList<Integer> B) {
        // write your code here
        int len = B.size();
        for (int i = 0; i < len; ++i)
            A.add(B.get(i));
        Collections.sort(A);
        return A;
    }
}

File: mu_wang/9_08/Merge Sorted Array II.py
class Solution:
    #@param A and B: sorted integer array A and B.
    #@return: A new sorted integer array
    def mergeSortedArray(self, A, B):
        # write your code here

        a = len(A) -1 
        b = len(B) -1 
        c = list()
        i,j=0,0
        while i<=a and j <= b :
            if A[i]<= B[j]:
                c.append(A[i])
                i+=1
            else:
                c.append(B[j])
                j+=1
        while i<=a:
            c.append(A[i])
            i+=1
        while j<=b:
            c.append(B[j])
            j+=1
        return c
            
            
        
